### ISSUE TITLE
SyntaxError: Invalid regular expression?

### ISSUE THREAD

**Issue by sweeneybros** [dansays/thingy#9]
Thingy sounds so awesome. I installed the action from actions.getdrafts.com and when I went to run it on a few test items I got this message

<img width="371" alt="Open Something" src="https://user-images.githubusercontent.com/58794248/70669791-3b76c680-1c2c-11ea-9fd5-57e17d1fde18.png">

I also get "?" instead of emoji's when trying to insert tags, project or area, notes, checklist items, etc. Only deadline and reminder seem to insert the correct emoji.

Hope I can get it to work!

**Comment by dansays** [comment]
Strange. Can you provide me with a copy of the draft you're attempting to process? And tell me a bit about your setup (Drafts version, iOS version, anything else of note)?

**Comment by sweeneybros** [comment]
I tried it with just one line that said "test" as well as a variety of
other lines of text. Always the same message regardless of what's written
in the list

iOS 13.2.2 on iPhone 6S+
Drafts v.16

Thanks for your help!

**Comment by dansays** [comment]
Very strange. I'm running on basically the same setup (on an iPhone 11, but that shouldn't matter) with no issues.

The app should automatically generate a "Thingy Autotagger Rules" draft. Any possibility you have a typo in there somewhere? If you delete the draft, it should regenerate the next time you run the "Add Task" action.

Have you tried uninstalling and re-installing the action?

**Comment by sweeneybros** [comment]
Tried uninstalling and re-installing the Thingy action. I checked for the
Thingy Autotagger Rules, and it's there. Same error re "invalid regular
expression: unrecognized character after (? - Line number:588, Column 31)".

I know this is just something your sharing for fun, so I'm expecting it to
be fixed or anything. Just wanted to know if this is an already-solved
issue that I can fix. Thanks

Ryan
626.260.3796

**Comment by dansays** [comment]
Yeah, if I can't repro it on my end, there's not much I can do. I would try deleting the Autotagger Rules draft and let it regenerate, on the off chance that you inadvertently made a change that's causing the error.

**Comment by horseninja** [comment]
Any further insight on this? I’m seeing the same error on iOS 13 and iPadOS. Also, the ?s instead of emojis for everything except for deadline and reminder. I’ve removed and reinstalled both Thingy and Drafts.

**Comment by horseninja** [comment]
I figured it out. In Drafts, you need to edit the individual actions, replacing the ?s with the correct emojis. 
For the attribute actions (tags, notes, etc.), open the action, then edit the Insert Text template and change the ? to the correct emoji. 
For the “Add Tasks to Things” action, the same emoji-for-? replacement needs to be done in the script where those are defined (lines in the mid-600s; the error seems to discount the commented-out lines at the beginning of the script).

**Comment by horseninja** [comment]
@sweeneybros ^^^
